conformance/buf.gen.yaml

```diff
@@ -20,6 +20,10 @@ plugins:
   - plugin: connect-kotlin
     out: google-javalite/build/generated/sources/bufgen
     path: ./protoc-gen-connect-kotlin/build/install/protoc-gen-connect-kotlin/bin/protoc-gen-connect-kotlin
+    opt:
```

Comment:
Updated the lite tests to use generated code with the same args as the full runtime tests.